swan-lake/resources/featured-scenarios/write-a-graphql-api-with-ballerina.md

```diff
@@ -8,177 +8,184 @@ active: write-a-graphql-api-with-ballerina
 
-The GraphQL endpoint you will create in this guide will have two main operations, `Query` and `Mutation`. The `Query` type will be used to read the data in the data source, and the `Mutation` operation will be used to update the data in the data source.
+The GraphQL endpoint you will create in this guide will have two primary operations, `Query` and `Mutation`. The `Query` type will be used to read the data in the data source, and the `Mutation` operation will be utilized to update the data in the data source.
 
 ### The `Query` type
 The `Query` type has two fields.
 * The `all` field - This field will return all the data in the data source as an array.
-* The `filter` field - This field will return the data filtered by the ISO Code of a country(`isoCode`).
+* The `filter` field - This field will return the data filtered by the ISO Code of a country(`iso_code`).
 
 ### The `Mutation` type
 The `Mutation` type has a single field.
 * The `add` field - This field will add a given entry to the data source.
 
-## Create the service package
 
-Ballerina uses packages to group code. You need to create a Ballerina package, generate the service code in the package, and write the business logic.
+## Create the service package 
 
->**Info:** For more information on Ballerina packages, see [Organize Ballerina code](/learn/organize-ballerina-code).
+Ballerina uses packages to group code. Follow the steps below to create a Ballerina package and write the business logic. 
 
-In the terminal, execute the command below to create the Ballerina package for the API implementation.
+> **Info:** For more information on Ballerina packages, see [Organize Ballerina code](/learn/organize-ballerina-code/).
 
-```
-$ bal new covid19 -t service
-```
-
-You view the output below.
-
-```
-Created new package 'covid19' at covid19.
-```
+1. In the Terminal, execute the command below to create the Ballerina package for the API implementation.
 
-This creates a directory named `covid19` with the default module along with a sample code for the service as shown below.
-
-```
-.
-├── covid19
-│   ├── tests
-│   │   └── service_test.bal
-│   ├── Ballerina.toml
-│   └── service.bal
-```
-
-## Create the data source 
+    ```
+    $ bal new write-a-graphql-api
+    ```
 
-Before writing the GraphQL service, let's create a data source for the project. This will mimic a database that stores the data for the service using an in-memory table in Ballerina as the data source.
+    You should see the output similar to the following.
 
-### Define the types for the data source
+    ```
+    package name is derived as 'write_a_graphql_api'. Edit the Ballerina.toml to change it.
```

Comment:
```suggestion
    package name is derived as 'write_a_graphql_api'. Edit the Ballerina.toml to change it.
```
Shouldn't this be outside the code block

Reply:
This is the output generated on the CLI.